index.html

```diff
@@ -0,0 +1,243 @@
+                        <li>
+                            <a href="index.html">
+                                Edu-Fair 2022
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <ul>
+                        <li>
+                            <a href="about.html">
+                                About
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                Program
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                Join
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                Sponsor
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                News
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="campaign">
+                    <ul>
+                        <li>
+                            <a href="index.html">
+                                CC Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <section class="event-info">
+            <h1>Hello All Aspiring Graduates!</h1>
+            <div class="highlight">
+                <h2>
+                    ACCRA EDU-FAIR 2022
+                </h2>
+            </div>
+            <div class="event-para">
+                <p class="event-para1">
+                    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Labore, numquam. Adipisci labore possimus atque recusandae excepturi porro incidunt tenetur distinctio.
+                </p>
+            </div>
+            <div>
+                <h3 class="event-date">
+                    2022-05-24 ~ 25
+                </h3>
+                <p class="event-para2">@ Accra International Conference Center</p>
+            </div>
+        </section>
+        <section class="program-schedule">
+            <h3 class="program-heading">Main Program</h3>
+            <span class="underline"></span>
+            <div class="program-outline">
+                <ul class="program">
+                    <li>
+                        <img src="images/icon1.png" alt="edu-fair icon">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
```

Comment:
- [ ] [Optional] Please, Adjust the alignments and padding as per given design. It's a best practice to follow given design `Margin, Padding, Alignments` as much as you can.